### PR TITLE
Add <uni-form-item-group> component

### PR DESCRIPTION
**Pull request by platosha**

### CODE REVIEW

File: components/uni-form-item/uni-form-item.html [hunk 1]
```diff
@@ -15,6 +15,18 @@
             display: none !important;
         }
 
+        /* Support <uni-form-item-group> to toggle a gap column in the item */
```
Comment (tomalec) [inline components/uni-form-item/uni-form-item.html]: Do we need gap column, can't we use grid-gap?
If so, I think it's not limited to `<uni-form-item-group>`, it's now part of public API (via CSS Var)

File: components/uni-form-item/uni-form-item.html [hunk 2]
```diff
@@ -28,6 +40,11 @@
             font-size: 12px;
         }
 
+        /* Enable border style overrides from `<uni-form-item-group>` */
```
Comment (tomalec) [inline components/uni-form-item/uni-form-item.html]: It's now API of this element, not necessarily for `uni-form-item-group`, I'd remove this element from comment, as in general, it's `/* Enable border style overrides from the outside */`. I'd add it to the README as a CSS custom prop available for customization.

File: components/uni-form-item/uni-form-item.html (hunk 1 above)
Comment (tomalec) [reply components/uni-form-item/uni-form-item.html]: Either use custom property in `:host` or even style it from the outside, like `uni-form-item{ grid-gap: xxx;}
`

Comment (tomalec) [reply components/uni-form-item/uni-form-item.html]: Let's keep this solution until we have `column-gap` for flex https://drafts.csswg.org/css-align-3/#gap-flex (CSS Box Alignment Module Level 3)

**Comment by tomalec** [comment]
I'm merging this to get the Edge fix sooner. Please add CSS Custom Properties to the README.md  in a separate PR.

**Comment by platosha** [comment]
@tomalec see https://github.com/Starcounter/uniform.css/pull/134 for the README.md updates

**Pull request merged**